msmbuilder/feature_selection/featureselector.py

```diff
@@ -0,0 +1,60 @@
+# Author: Carlos Xavier Hernandez <cxh@stanford.edu>
+# Contributors:
+# Copyright (c) 2016, Stanford University and the Authors
+# All rights reserved.
+import numpy as np
+
+from .base import MultiSequenceFeatureSelectionMixin
+
+
+class FeatureSelector(MultiSequenceFeatureSelectionMixin):
```

Comment:
I'll make a note in the docstring, but I think a warning might be a tad much. @mpharrigan thoughts?